internal/tjson/tjson_test.go

```diff
@@ -222,28 +229,38 @@ func testJSON(t *testing.T, testCases []testCase, newFunc func() tjsontype) {
 
 func fuzzJSON(f *testing.F, testCases []testCase) {
 	for _, tc := range testCases {
-		f.Add(tc.j)
+		schema, err := tc.schema.Marshal()
+		if err != nil {
+			f.Fatalf("failed to marshal schema: %v", err)
```

Comment:
Why not `require.NoError(f, err)` as we do usually?

Reply:
No reason, it is left after debugging.